Core/MemMapFunctions.cpp

```diff
@@ -87,7 +87,7 @@ inline void ReadFromHardware(T &var, const u32 address) {
 		var = *((const T*)GetPointerUnchecked(address));
 	} else {
 		// In jit, we only flush PC when bIgnoreBadMemAccess is off.
-		if (g_Config.bJit && g_Config.bIgnoreBadMemAccess) {
+		if (g_Config.iCpuCore != CPU_CORE_INTERPRETER && g_Config.bIgnoreBadMemAccess) {
```

Comment:
I think this can be only when in jit.

Really, this is just so that lw/sw/etc. will not log a completely wrong PC and confuse people.  We don't flush the PC in slowmem when calling these funcs.

We do flush before syscalls, and we only use unchecked in IR interpret.  I wonder what this would be set to in a IR-based jit...

Ideally we'd still log the address in syscalls anyway, maybe it'd be better to have a separate func...

-[Unknown]